.github/workflows/containers.yml

```diff
@@ -103,10 +103,18 @@ jobs:
                     APIML_SECURITY_OIDC_CLIENTSECRET: ${{ secrets.OKTA_CLIENT_PASSWORD }}
                     APIML_SECURITY_OIDC_INTROSPECTENDPOINT: /v1/introspect
                     APIML_SECURITY_OIDC_ENABLED: true
+                    APIML_SECURITY_OIDC_REGISTRY: zowe.okta.com
+                    APIML_SECURITY_X509_EXTERNALMAPPERUSER: mockUser
+                    APIML_SECURITY_X509_EXTERNALMAPPERURL: https://mock-services:10013/zss/api/v1/certificate
```

Comment:
in real setup, traffic would go through the gateway so there should be gateway url

Reply:
duplicate

Reply:
Resolved. Mapper is now called through the gateway.